app/components/contributor_form/contributor_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with(model: contributor, local: true, class: "ContributorForm", multipart: true) do %>
+<%= form_with(model: contributor, local: true, class: "ContributorForm", multipart: true, data: { turbo: false }) do %>
```

Comment:
Why’s this necessary?

Reply:
That's something I caught along the way: Flash messages (validation errors) won't be displayed otherwise.